invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -122,6 +123,29 @@ <h2>{{ add_description.type.title_l10n }} <span
           {%- endfor -%}
         {%- endif -%}
       </dt>
+    {%- else -%}
```

Comment:
Maybe just put the title section in the if? That way you could avoid the copied code.

Reply:
Hi @tmorrell I modified the code, can you have another look?

Reply:
That looks good!